doc/whats_new/v0.20.rst

```diff
@@ -228,6 +228,9 @@ Preprocessing
 - Updated :class:`preprocessing.MinMaxScaler` to pass through NaN values.
   :issue:`10404` by :user:`Lucija Gregov <LucijaGregov>`.
 
+- Updated :func:`preprocessing.minmax_scale` to pass through NaN values.
```

Comment:
I think it's more helpful to users if this entry is combined with the previous.

Reply:
good point